### DISCUSSION TITLE
TasmoAdmin via https?

### DISCUSSION THREAD

**Discussion by nashaden** [TasmoAdmin/TasmoAdmin#856] [Q&A]
So I've recently switched to Traefik+Let's Enrcypt to access all my internal services that run in docker (and also as bare metal) via https. In the installation guide, only port 80 is mentioned. Is it possible to set it up to use https?

**Comment by inverse** [comment]
Check https://github.com/TasmoAdmin/TasmoAdmin/issues/730

The problem with HTTPS is that Tasmota devices won't be able to update via that so we would need to serve the firmware directory from HTTP and the rest via HTTPS. It's possible but would need some work
